diff -cr sgml-tools-1.0.9-orig/lib/SGMLTools/Lang.pm sgml-tools-1.0.9/lib/SGMLTools/Lang.pm
*** sgml-tools-1.0.9-orig/lib/SGMLTools/Lang.pm	Sat May 16 12:28:59 1998
--- sgml-tools-1.0.9/lib/SGMLTools/Lang.pm	Sun Dec  3 10:33:57 2000
***************
*** 60,65 ****
--- 60,66 ----
    pt portuges portuguese
    ca català catalan
    it italiano italian
+   pl polski polish
    ro românã romanian
  );
  
***************
*** 164,169 ****
--- 165,171 ----
       "pt" => "Página anterior",
       "ca" => "Pàgina anterior",
       "it" => "Indietro",
+      "pl" => "Poprzednia",
       "ro" => "Înapoi"
    },
    "Next" => {
***************
*** 177,182 ****
--- 179,185 ----
       "pt" => "Página seguinte",
       "ca" => "Pàgina següent",
       "it" => "Avanti",
+      "pl" => "Nastêpna",
       "ro" => "Înainte"
    },
    "Contents" => {
***************
*** 190,195 ****
--- 193,199 ----
       "pt" => "Índice",
       "ca" => "Índex",
       "it" => "Indice",
+      "pl" => "Spis tre¶ci",
       "ro" => "Cuprins"
    },
    "Table of Contents" => {
***************
*** 203,208 ****
--- 207,213 ----
       "pt" => "Índice geral",
       "ca" => "Índex general",
       "it" => "Indice Generale",
+      "pl" => "Spis tre¶ci",
       "ro" => "Cuprins"
    }
  };
diff -cr sgml-tools-1.0.9-orig/lib/dist/sgmltool/latex2e/mapping sgml-tools-1.0.9/lib/dist/sgmltool/latex2e/mapping
*** sgml-tools-1.0.9-orig/lib/dist/sgmltool/latex2e/mapping	Fri Jun 26 21:39:17 1998
--- sgml-tools-1.0.9/lib/dist/sgmltool/latex2e/mapping	Sun Dec  3 10:37:53 2000
***************
*** 8,16 ****
  			"\\usepackage{linuxdoc-sgml}\n"
  			"\\usepackage{qwertz}\n"
  			"\\usepackage{url}\n"
! 			"\\usepackage\[latin1\]{inputenc}\n"
  			"\\usepackage{t1enc}\n"
! 			"\\usepackage{babel}\n"
  			"\\usepackage{epsfig}\n"
  			"\\usepackage{[OPTS]}\n"
  			"\\def\\addbibtoc{\n"
--- 8,16 ----
  			"\\usepackage{linuxdoc-sgml}\n"
  			"\\usepackage{qwertz}\n"
  			"\\usepackage{url}\n"
! 			"\\usepackage\[latin2\]{inputenc}\n"
  			"\\usepackage{t1enc}\n"
! 			"\\usepackage\[polish\]{babel}\n"
  			"\\usepackage{epsfig}\n"
  			"\\usepackage{[OPTS]}\n"
  			"\\def\\addbibtoc{\n"
***************
*** 23,31 ****
  			"\\usepackage{linuxdoc-sgml}\n"
  			"\\usepackage{qwertz}\n"
  			"\\usepackage{url}\n"
! 			"\\usepackage\[latin1\]{inputenc}\n"
  			"\\usepackage{t1enc}\n"
! 			"\\usepackage{babel}\n"
  			"\\usepackage{epsfig}\n"
  			"\\usepackage{[OPTS]}\n"
  			"\\def\\addbibtoc{\n"
--- 23,31 ----
  			"\\usepackage{linuxdoc-sgml}\n"
  			"\\usepackage{qwertz}\n"
  			"\\usepackage{url}\n"
! 			"\\usepackage\[latin2\]{inputenc}\n"
  			"\\usepackage{t1enc}\n"
! 			"\\usepackage\[polish\]{babel}\n"
  			"\\usepackage{epsfig}\n"
  			"\\usepackage{[OPTS]}\n"
  			"\\def\\addbibtoc{\n"
***************
*** 39,47 ****
  			"\\usepackage{linuxdoc-sgml}\n"
  			"\\usepackage{qwertz}\n"
  			"\\usepackage{url}\n"
! 			"\\usepackage\[latin1\]{inputenc}\n"
  			"\\usepackage{t1enc}\n"
! 			"\\usepackage{babel}\n"
  			"\\usepackage{epsfig}\n"
  			"\\usepackage{[OPTS]}\n"
  			"\\def\\addbibtoc{\n"
--- 39,47 ----
  			"\\usepackage{linuxdoc-sgml}\n"
  			"\\usepackage{qwertz}\n"
  			"\\usepackage{url}\n"
! 			"\\usepackage\[latin2\]{inputenc}\n"
  			"\\usepackage{t1enc}\n"
! 			"\\usepackage\[polish\]{babel}\n"
  			"\\usepackage{epsfig}\n"
  			"\\usepackage{[OPTS]}\n"
  			"\\def\\addbibtoc{\n"
***************
*** 56,64 ****
  			"\\usepackage{linuxdoc-sgml}\n"
  			"\\usepackage{qwertz}\n"
  			"\\usepackage{url}\n"
! 			"\\usepackage\[latin1\]{inputenc}\n"
  			"\\usepackage{t1enc}\n"
! 			"\\usepackage{babel}\n"
  			"\\usepackage{epsfig}\n"
  			"\\usepackage{[OPTS]}\n"
  			"\\def\\title#1{\n"
--- 56,64 ----
  			"\\usepackage{linuxdoc-sgml}\n"
  			"\\usepackage{qwertz}\n"
  			"\\usepackage{url}\n"
! 			"\\usepackage\[latin2\]{inputenc}\n"
  			"\\usepackage{t1enc}\n"
! 			"\\usepackage\[polish\]{babel}\n"
  			"\\usepackage{epsfig}\n"
  			"\\usepackage{[OPTS]}\n"
  			"\\def\\title#1{\n"
